docs/content/docs/getting-started/nuxt.mdx

```diff
@@ -55,25 +55,24 @@ export default defineNuxtConfig({
 });
 ```
 
+This will also automatically enable the TypeScript plugin, which provides helpful IntelliSense hints in your IDE for workflow and step functions.
+
 <Accordion type="single" collapsible>
-  <AccordionItem value="typescript-intellisense" className="[&_h3]:my-0">
+  <AccordionItem value="typescript-plugin-config" className="[&_h3]:my-0">
```

Comment:
hmm I would like to stay consistent with other docs on these for deeplinking

Reply:
of course 👍